akamas/README.md

```diff
@@ -0,0 +1,40 @@
+# Akamas
+
+## Overview
+
+Akamas Insights helps developers, DevOps engineers, and SREs quickly identify cost inefficiencies and reliability issues in Kubernetes environments. It collects observability data from Datadog and analyzes workloads, application runtimes like the JVM and cluster behaviour to automatically identify optimization opportunities and generate easy-to-apply recommendations.
+
+**Cost Efficiency Analysis**
+detect issues like over-provisioned workloads, over-sized JVM heap memory, or inefficient cluster autoscaling behavior and estimate savings, so that you can focus on the biggest cost reduction opportunities.
```

Comment:
```suggestion
Detect issues like over-provisioned workloads, over-sized JVM heap memory, or inefficient cluster autoscaling behavior and estimate savings, so that you can focus on the biggest cost reduction opportunities.
```